File: C#/7kyu/Binary Addition/KataTest.cs
using System;
using NUnit.Framework;
using Kata;

namespace Tests
{
    [TestFixture]
    public class AddBinaryTest
    {
        [Test]
        public void TestExample()
        {
            Assert.AreEqual("11", KataClass.AddBinary(1, 2), "Should return \"11\" for 1 + 2");
        }
    }
}

File: JavaScript/6kyu/Playing with digits/KataTest.js
const Test = require("tape");
const Kata = require("./Kata.js");

Test("should work for some examples", (assert) => {
    assert.equal(Kata.digPow(89, 1), 1)
    assert.equal(Kata.digPow(92, 1), -1)
    assert.equal(Kata.digPow(46288, 3), 51)
    assert.end();
})